unity-renderer/Assets/ABConverter/AssetBundleMenuItems.cs

```diff
@@ -45,9 +45,10 @@ public class EmptyParcels
 
         public static void DumpEmptyParcels(string folderName = "empty-scenes")
         {
+            // The paths in this method work if the unity-renderer repo is at the same location as the explorer repo
             string indexJsonPath = Application.dataPath;
 
-            indexJsonPath += $"/../../kernel/static/loader/{folderName}/index.json";
+            indexJsonPath += $"/../../../explorer/kernel/static/loader/{folderName}/index.json";
```

Comment:
As this is a local path and should be set manually each time we want to use it, I'd add a constant with the `/../../../explorer/kernel` bit, and clarify that in a comment.

Reply:
Addressed, check it out!